services/spar/test/Arbitrary.hs

```diff
@@ -46,7 +46,7 @@ instance Arbitrary NoContent where
   arbitrary = pure NoContent
 
 instance Arbitrary IdPMetadataInfo where
-  arbitrary = IdPMetadataValue <$> arbitrary
+  arbitrary = IdPMetadataValue mempty <$> arbitrary
```

Comment:
```suggestion
  arbitrary = do
    xml <- arbitrary
    pure $ IdPMetadataValue (encode xml) xml
```

How about this?

Reply:
good idea!  and it *almost* type-checked.  :)

one more commit coming up...